communities/hacs/readme.md

```diff
@@ -0,0 +1,38 @@
+[![Build Status](https://travis-ci.org/HaCSBCU/hacsbcu.github.io.svg?branch=master)](https://travis-ci.org/HaCSBCU/hacsbcu.github.io)
+
+![Website Screenshot](http://i.imgur.com/lJnqm1B.jpg)
```

Comment:
On the image thing I've discovered the trick of posting in an issue to get the github cdn link, so its all within github and doesn't litter your repo.

I used that trick on the awesome repo and discuss